### DISCUSSION TITLE
Активы компании и постановка задач в организации

### DISCUSSION THREAD

**Discussion by Davud77** [totumonline/totum-ru-issues-and-discussions#117] [Сделанные проекты]
Доброго времени суток!

У нас работает система учета активов компании.
Мы не программисты, изначально всю работу вели в SharePoint, после перешли в Google Таблицы.
Путем длительных поисков вышли на TOTUM

Активы:
![image](https://user-images.githubusercontent.com/26651156/164623221-ee132f3e-2439-4ce3-8e85-b92574358411.png)

Карточка актива:
![image](https://user-images.githubusercontent.com/26651156/164623710-f0cd0c26-747a-483d-bcf9-bedb938a6d4f.png)

Задачи:
![image](https://user-images.githubusercontent.com/26651156/164624991-2bb892db-6b59-473a-92cb-e715acfd82a0.png)

Очень понравилось в Totum: 
Наличие русской документации;
Наличие демо для тренировки;
Огромное количество поясняющих роликов на YouTube;
Практически моментальная связь с разработчиком в telegram.

**Comment by ttm-alexeypolunin** [comment]
Круто, а как вы такой черный сделали — это авторежим браузера?

**Reply by Davud77** [reply]
Расширение для браузера - Dark Reader

**Reply by ttm-alexeypolunin** [reply]
Спс! Я попробую на выходных, тк то, что я пробовал до этого съедало часть цветов и становилось непонятно.

**Reply by Qb1cc** [reply]
Можно же отредачить ваш CSS , но проблема в том что при обновлении приходится файлик редаченый сохранять куда то , а потом опять запихивать обратно.  
Просили передвинуть кнопки, передвинул с помощью css

![image](https://user-images.githubusercontent.com/84081305/165039182-cbdfd1c2-3dc4-4e27-8eda-09480b6265de.png)

**Reply by ttm-alexeypolunin** [reply]
Учтите, что левая сторона используется под `комментарии строчной части таблицы`.  С точки зрения Git это как-то делается — надо поэксперементировать. Или может кто здесь подскажет, но это вряд ли.
